librz/bin/format/mach0/mach0.c

```diff
@@ -1537,6 +1536,7 @@ static bool reconstruct_chained_fixup(struct MACH0_(obj_t) * bin) {
 	if (!bin->nsegs) {
 		return false;
 	}
+	bin->nchained_starts = bin->nsegs;
 	bin->chained_starts = RZ_NEWS0(struct rz_dyld_chained_starts_in_segment *, bin->nsegs);
```

Comment:
```suggestion
	bin->chained_starts = RZ_NEWS0(struct rz_dyld_chained_starts_in_segment *, bin->nchained_starts);
```
Just a small change to express the meaning of `nchained_starts` better.